Doc/tools/extensions/pydoc_topics.py

```diff
@@ -102,6 +103,14 @@
     "yield",
 })
 
+class PydocTextTranslator(TextTranslator):
+    def visit_grammar_snippet(self, node: Element) -> None:
```

Comment:
Add `from xml.dom.minidom import Element` into the `if TYPE_CHECKING:` block for this?

Reply:
It's `nodes.Element` :)